package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
```

Comment:
jshint looks for 4 spaces per indent, that file was not compliment with that requirement.